app/views/articles/_related_videos.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="youtube-videos">
+  <%= render "liquids/youtube", id: "rvfmPtkhqok", width: 766, height: 431 %>
+  <%= render "liquids/youtube", id: "-Mkp4oavz2M", width: 766, height: 431 %>
+  <%= render "liquids/youtube", id: "0Atk61qbLT0", width: 766, height: 431 %>
+</div>
+<script>
+  const options = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    }
+  }
+  fetch(`http://localhost:3000/relevant_content?keywords=westworld`)
```

Comment:
Also, I think there might be an ENV variable we can use to make this dynamic and usable in production. Ok for now since we aren't required to deploy to Heroku

Reply:
Or wait, give this a try without `http://localhost:3000`
It might work with just `/relevant_content?keywords=westworld`